envcontext.js

```diff
@@ -491,6 +506,80 @@ export function installSdkmanTool(toolType, toolName) {
   return true;
 }
 
+/**
+ * Method to check if a given nvm tool is installed and available.
+ *
+ * @param {String} toolName Tool name with version. Eg: 22.0.2-tem
+ *
+ * @returns {String} path of nvm if present, otherwise false
+ */
+export function ifNvmToolAvailable(toolName) {
```

Comment:
Can we call this method as `getNvmToolDirectory` since this is not a boolean method?